cypress/integration/customizer/general/buttons.spec.js

```diff
@@ -0,0 +1,115 @@
+let colors = {
+	hexaBackground: '#38d421',
+	hexaText: '#830202',
+	rgbBackground: 'rgb(56, 212, 33)',
+	rgbText: 'rgb(131, 2, 2)'
+}
+describe('Primary button control ', function () {
+
+	it('Changes button settings in customizer', function () {
+		cy.login();
+		cy.visit('/wp-admin/customize.php');
+		cy.get('#accordion-section-neve_buttons_section').click();
+
+		//Changes border-radius
+		cy.get('#customize-control-neve_button_appearance > .neve-button-appearance-control > .neve-white-background-control > .neve-responsive-sizing > :nth-child(2) > .top-input').
+		click().clear().
+		//the border radius will be 20
+		type('2');
+
+		//Changes the background color
+		cy.get('button.components-button.is-secondary.is-button').
+		filter(":visible").
+		eq(0).
+		click();
+		cy.get('input.components-text-control__input').click({
+			force: true
+		}).clear().type(colors.hexaBackground);
+
+		//Changes text color
+		cy.get('button.components-button.is-secondary.is-button').
+		filter(":visible").
+		eq(1).
+		click();
+		cy.get('input.components-text-control__input').click({
+			force: true
+		}).clear().type(colors.hexaText);
+
+		cy.get('button.components-button.components-panel__body-toggle').
+		contains('Normal').click();
+
+		//Change hover colors, can't check it in front-end till cypress make it possible.
+		/*
+		cy.get('button.components-button.components-panel__body-toggle').
+		contains('Hover').click();
+
+		cy.get('div.neve-control-header.neve-color-component').
+		filter(':visible').find('button').eq('0').click();
+		cy.get('input.components-text-control__input').click({ force: true }).clear().type('#eeee13');
+		
+		cy.get('div.neve-control-header.neve-color-component').
+		filter(':visible').find('button').eq('4').click();
+		cy.get('input.components-text-control__input').click({ force: true }).clear().type('#dc0adc');
+		*/
+
+		cy.get('#save').click();
+		cy.wait(2000);
```

Comment:
Is this wait necessary? It seems that you're trying to prep something for the next text, it would be better to set this up on the next text instead on a something like after hook inside the test. Check Cypress documentation regarding setting up state for test: https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks